content/en/references/_index.adoc

```diff
@@ -7,8 +7,6 @@ description: >
   Technical documents relevant to the project.
 ---
 
-* https://docs.google.com/document/d/1WbaJ5lhCMrui6hDj-EqaMEB5Ymmw-CQgvQznhYMCw9w/edit?usp=sharing[Project Proposal]
-* https://docs.google.com/document/d/164DD9j-OUmsBKB3FELcanB-5L1GkFy0qX10EiUfEtjM/edit[Testing Questions]
-* https://docs.google.com/spreadsheets/d/1FWTYRxglWDqIK_xI6HlBGKUegICQRsXe5NHCJTlTlHs/edit?usp=sharing[Printing Queue]
-* https://docs.google.com/presentation/d/1UeSRyUB0dAOZZecX4TGh40GkVtGADlXzBOwVFjPqIFE/edit#slide=id.g7f621674bc_0_63[System Concept Diagram]
-* link:circuit_schematic.adoc[Circuit Schematic UML Diagram]
+link:ibd-system-electronic-sensing-system-dataflow.adoc[Sensing System UML Diagram]
+link:ibd-system-ventilator-splitter-assembly-airflow.adoc[Splitter Assembly UML Diagram]
+link:hardware_requirements.adoc[Hardware Requirements List]
```

Comment:
You can actually remove these links (line 10-12), since Docsy will create links to the nested pages automatically.